e2e/miniatures/test_themes_crud.py

```
@@ -0,0 +1,306 @@
+            print("  [PASS] Update theme data")
+            print("  [PASS] Data persistence after reload")
+            print("  [PASS] Search by name")
+            print("  [PASS] Delete theme")
+            print("  [PASS] Verify deletion")
+            print("  [PASS] Deletion persistence")
+            print("\nScreenshots saved to /tmp/:")
+            for i in range(1, 12):
+                if i != 7 and i != 8 and i != 9 and i != 11:  # Skip steps without screenshots
+                    print(f"  - themes_{i:02d}_*.png")
+
+            return True
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)